laws/src/main/scala/cats/laws/NonEmptyParallelLaws.scala

```diff
@@ -13,6 +13,9 @@ trait NonEmptyParallelLaws[M[_], F[_]] {
 
   def sequentialRoundTrip[A](fa: F[A]): IsEq[F[A]] =
     P.parallel(P.sequential(fa)) <-> fa
+
+  def isomorphicFunctor[A, B](fa: F[A], f: A => A): IsEq[M[A]] =
```

Comment:
Why `A=> A` and not B? If we keep just A we should remove the B type parameters.